-- Remove the CMP0048 OLD policy that CMake 4.5 no longer supports.
-- Enable PIC on all amd64 UNIX systems, not only those where CMake reports
-- CMAKE_SYSTEM_PROCESSOR as x86_64 (FreeBSD reports AMD64).

--- CMakeLists.txt.orig	2026-10-05 16:26:15 UTC
+++ CMakeLists.txt
@@ -32,12 +32,6 @@ cmake_minimum_required(VERSION 2.6)
 project(qucs-core CXX)
 cmake_minimum_required(VERSION 2.6)
 
-# ignore the project() managed VERSION (new in CMake 3.0)
-if(POLICY CMP0048)
-  cmake_policy(SET CMP0048 OLD)
-endif(POLICY CMP0048)
-
-
 # otherwise qucsator cannot generate qucsdefs.h
 add_definitions(-DDEBUG)
 
@@ -276,9 +270,7 @@ if(UNIX AND NOT APPLE)
 # Set position independent code PIC
 #
 if(UNIX AND NOT APPLE)
-  if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
-    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-  endif()
+  set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 endif()
 
 #
